python/examples/ola_devices.py

```diff
@@ -21,6 +21,9 @@
 __author__ = 'nomis52@gmail.com (Simon Newton)'
 
 from ola.ClientWrapper import ClientWrapper
+from ola.OlaClient import RequestStatus
```

Comment:
Are the changes here and below okay @nomis52 , before I apply them to the other files and we can start getting some of this merged in?